bridge_adaptivity/module/models.py

```diff
@@ -104,25 +121,16 @@ class GradingPolicy(ModelFieldIsDefaultMixin, models.Model):
 
     name = models.CharField(max_length=20)
     public_name = models.CharField(max_length=255)
-    threshold = models.PositiveIntegerField(blank=True, default=0, help_text="Grade policy: 'Q'")
+    threshold = models.PositiveIntegerField(blank=True, default=1, help_text="Grade policy: 'Q'")
     is_default = models.BooleanField(default=False)
 
-    def _points_earned_grade(self, trials_count, points_earned):
-        return points_earned / max(self.threshold, trials_count)
-
-    def _trials_count(self, trials_count):
-        return trials_count / max(self.threshold, trials_count)
+    @property
+    def policy_cls(self):
+        return GRADING_POLICY_NAME_TO_CLS[self.name]
```

Comment:
Please add a comment that `self.name` if fulfilled from the correspondent `choices` in the form.